ubuntu-apache-test-page/azuredeploy.json

```diff
@@ -89,22 +89,22 @@
     "vhdStorageContainerName": "vhds",
     "vmName": "[concat('webtestvm-', parameters('dnsNameForPublicIP'), '-', uniqueString(resourceGroup().id))]",
     "vmSize": "Standard_D2_v2",
-    "virtualNetworkName": "[concat('webtestvnet-', uniqueString(resourceGroup().id))]",
+    "virtualNetworkName": "[concat('webtestvnet-', uniqueString(resourceGroup().id, parameters('dnsNameForPublicIP')))]",
```

Comment:
the extra param on uniqueString doesn't seem necessary, is there a reason why you added it?